src/components/report-problem.style.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+export default {
+  camera: {
```

Comment:
Yeah, that would be great.  I mean pretty much everything is flex:1.  Let me do that.